lib/bugsnag/report.rb

```diff
@@ -51,9 +54,12 @@ def initialize(exception, passed_configuration, auto_notify=false)
       self.api_key = configuration.api_key
       self.app_type = configuration.app_type
       self.app_version = configuration.app_version
+      self.breadcrumbs = []
       self.delivery_method = configuration.delivery_method
       self.hostname = configuration.hostname
+      self.message = defined?(exception.message) ? exception.message : exception.to_s
       self.meta_data = {}
+      self.name = exception.class.to_s
```

Comment:
If `exception` is a string (which is likely true if `exception` doesn't respond to `message`) instead of an exception, would it make more sense to default to `RuntimeError` instead (because code later on will convert it to that)?

Reply:
I think the breadcrumb name should reflect what was notified explicitly.  In this case the result would be `String` which on a breadcrumb with limited information I think is more telling than `RuntimeError` would be.

Reply:
Users will use this information to try and find the error in the UI so switching between class and message will be confusing.

Reply:
That's fair, I hadn't considered it in that context.